### ISSUE TITLE
Running HBUP on a Mac and accessing an livestream from the Home app on the same Mac results in being unable to initiate video stream.

### ISSUE THREAD

**Issue by LokiThalhammer** [hjdhjd/homebridge-unifi-protect#907]
**Describe The Problem:**

Unable to stream video in Home app on Mac Studio. I can stream on MacBook Pro, iPhone, iPad while on the same network. Preview updates as expected, playback of motion events within the Home app also work fine. **Note:** Homebridge is running on the same mac that is having the issue. 

Tested new user
Tried low quality
Enabled Transcoding
Enabled Transcoding Hardware
Removed Homebridge and Accessories from home, re added
Removed and reinstalled FFmpeg 
Change Network network interface
Changed mDNS  

**To Reproduce:**

Open Home > Click preview of camera feed > Spinning Gear > Nothing

**Logs:**

[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: Streaming request from 192.168.1.93: 1280x720@30fps, 299 kbps. Using 1920x1080@30fps (High), 6000 kbps. Audio packet time = 30
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: FFmpeg command: /usr/local/lib/node_modules/homebridge-unifi-protect/node_modules/ffmpeg-for-homebridge/ffmpeg -hide_banner -probesize 32768 -max_delay 500000 -r 30 -rtsp_transport tcp -i rtsps://192.168.1.1:7441/DGisQAqDnuolljdi?enableSrtp -map 0:v:0 -vcodec copy -payload_type 99 -ssrc 15974576 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params N13MRvaTKmeBgbbcMtMkehRXTtu68AoTPyE8ugcB srtp://192.168.1.93:52227?rtcpport=52227&localrtcpport=52227&pkt_size=564 -map 0:a:0 -acodec libfdk_aac -profile:a aac_eld -flags +global_header -f null -ar 24k -afterburner 1 -eld_sbr 1 -eld_v2 1 -b:a 24k -bufsize 48k -ac 1 -payload_type 110 -ssrc 3381634 -f rtp -srtp_out_suite AES_CM_128_HMAC_SHA1_80 -srtp_out_params QDUieks/iHY8sHfk0iBYsRIKQLYn9v1We6FXc9C/ srtp://192.168.1.93:53665?rtcpport=53665&localrtcpport=53665&pkt_size=188 -loglevel level+verbose
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [tcp @ 0x600003748090] [verbose] Starting connection attempt to 192.168.1.1 port 7441
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [tcp @ 0x600003748090] [verbose] Successfully connected to 192.168.1.1 port 7441
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [rtsp @ 0x13a604660] [verbose] SDP:
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: v=0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: o=- 604 0 IN IP4 192.168.1.1
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: s=7483C28F22D3_0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: u=www.evostream.com
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: e=contact@evostream.com
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: c=IN IP4 192.168.1.1
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: t=0 0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=recvonly
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=control:*
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=range:npt=now-
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: m=audio 0 RTP/AVP 96
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=recvonly
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=rtpmap:96 mpeg4-generic/11025/1
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=control:trackID=0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=fmtp:96 streamtype=5; profile-level-id=15; mode=AAC-hbr; config=1508; SizeLength=13; IndexLength=3; IndexDeltaLength=3;
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=crypto:1 AES_CM_128_HMAC_SHA1_80 inline:+50CTcqxz21kDX8LhJdNBHOuVFkE/792R7YESgN1
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: m=audio 0 RTP/AVP 96
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=recvonly
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=rtpmap:96 opus/48000/2
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=control:trackID=1
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=fmtp:96
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=crypto:1 AES_CM_128_HMAC_SHA1_80 inline:AsAjYmIROC4nRUAx4tLbOI9JHAK5AWwgaIMvUS3W
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: m=video 0 RTP/AVP 97
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=recvonly
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=control:trackID=2
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=rtpmap:97 H264/90000
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=fmtp:97 profile-level-id=4d0029; packetization-mode=1; sprop-parameter-sets=Z00AKZpkA8ARPy/+AXoBfNwEBAUAAAMD6AAA6mCdoIhGoA==,aO48gA==
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: a=crypto:1 AES_CM_128_HMAC_SHA1_80 inline:tPDGOFL/TQDPG4gTIN+xThP6TxLLTsBAVVm9Pfmf
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [rtsp @ 0x13a604660] [verbose] setting jitter buffer size to 0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [rtsp @ 0x13a604660] [verbose] setting jitter buffer size to 0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [rtsp @ 0x13a604660] [verbose] setting jitter buffer size to 0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [h264 @ 0x13c0057c0] [verbose] Reinit context to 1920x1088, pix_fmt: yuvj420p
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [rtsp @ 0x13a604660] [warning] Stream #2: not enough frames to estimate rate; consider increasing probesize
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info] Input #0, rtsp, from 'rtsps://192.168.1.1:7441/DGisQAqDnuolljdi?enableSrtp':
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info]   Metadata:
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info]     title           : 7483C28F22D3_0
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info]   Duration: N/A, start: 0.000000, bitrate: N/A
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info]   Stream #0:0: Audio: aac, 11025 Hz, mono, fltp
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info]   Stream #0:1: Audio: opus, 48000 Hz, stereo, fltp
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [info]   Stream #0:2: Video: h264, 1 reference frame, yuvj420p(pc, bt709, progressive, left), 1920x1080 (1920x1088) [SAR 189:190 DAR 168:95], 30 fps, 30 tbr, 90k tbn
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [udp @ 0x6000032502c0] [error] bind failed: Address already in use
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: [error] srtp://192.168.1.93:52227?rtcpport=52227&localrtcpport=52227&pkt_size=564: Input/output error
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: The FFmpeg process ended unexpectedly due to issues with the media stream provided by the UniFi Protect livestream API. Adjusting the settings we use for FFmpeg temporarily to use safer values at the expense of some additional streaming startup latency.
[3/30/2023, 10:38:31 PM] [UniFi Protect] Dream Machine Pro [UDM-PRO] Back Door [G3 Bullet]: Stopped video streaming session.

**Homebridge Configuration:**

```json
{
    "bridge": {
        "name": "Homebridge 79E9",
        "username": "0E:BE:EA:DE:79:E9",
        "port": 51725,
        "pin": "610-97-527",
        "advertiser": "bonjour-hap",
        "bind": [
            "en1"
        ]
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Home",
            "port": 8581,
            "auth": "form",
            "theme": "grey",
            "tempUnits": "c",
            "lang": "en",
            "platform": "config"
        },
        {
            "controllers": [
                {
                    "address": "192.168.1.1",
                    "password": "HXXXX",
                    "username": "XXXXX"
                }
            ],
            "name": "UniFi Protect",
            "verboseFfmpeg": true,
            "platform": "UniFi Protect"
        }
    ],
    "disabledPlugins": []
}
```

**Screenshots:**

NA

**Environment:**

* **Homebridge Version:** 1.6.0
* **Node Version:** 18.15.0
* **Homebridge-UniFi-Protect Plugin Version:** 6.0.3 
* **UniFi Protect Version:** Protect 2.2.11
* **UniFi Protect Controller Type:** UDMP
* **UniFi Protect Controller Firmware Version:** 2.5.17
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** Mac Studio with macOS
* **Operating System and OS Version:** Ventura 13.3 (22E252)

**Comment by hjdhjd** [comment]
Known Homebridge issue in general on Macs when trying to run video streams from the Home app on the same machine you're running Homebridge. It's on my backlog...I'm making incremental headway here, but it's not going to get solved in the immediate future.

Appreciate it and apologies...

**Comment by LokiThalhammer** [comment]
Thanks for the quick response, no apologies needed. Now I know what it is, I can stop stressing and wait patiently. Thanks again for all you do!

**Comment by hjdhjd** [comment]
This will be fixed in the next release. Thanks again for re-raising an age-old problem that needed solving.

**Comment by hjdhjd** [comment]
Should be fixed in v6.1.0. Thanks!

**Comment by LokiThalhammer** [comment]
I can confirm this is now fixed. Thank you, I'm really impressed with the overall experience. Cheers!
